.golangci.yml

```diff
@@ -39,5 +39,8 @@ issues:
     - path: "(.+)_test.go"
       linters:
         - gosec
+    - path: "internal/config/config.go"
```

Comment:
Go sec is reporting hardcoded credentials on:

```
const providerName = "Terraform-Provider-Auth0"
```

Reply:
Can we add inline w/ code to prevent exclusion of the entire file?

Reply:
Updated in [`f81d81a` (#789)](https://github.com/auth0/terraform-provider-auth0/pull/789/commits/f81d81ab0e2f1a45ea904b2bcae81040b0a65d57)